src/main/java/org/zeroturnaround/zip/ZipUtil.java

```diff
@@ -955,6 +955,58 @@ public void process(InputStream in, ZipEntry zipEntry) throws IOException {
       }
     }
   }
+  
+  /**
+   * Unpacks each ZIP entries. Presumes they are packed with the backslash separator.
+   * Some archives can have this problem if they are created with some software
+   * that is not following the ZIP specification.
+   */
+  protected static class BackslashUnpacker implements ZipEntryCallback {
+
+    private final File outputDir;
+    private final NameMapper mapper;
+
+    public BackslashUnpacker(File outputDir, NameMapper mapper) {
+      this.outputDir = outputDir;
+      this.mapper = mapper;
+    }
+
+    public void process(InputStream in, ZipEntry zipEntry) throws IOException {
+      String name = mapper.map(zipEntry.getName());
+      if (name != null) {
+        /**
+         * We assume that EVERY backslash will denote a directory
+         * separator. Also such broken archives don't have entries that
+         * are just directories. Everything is a file. See the example
+         *                                                                                                                                                                                                   1 ↵
```

Comment:
Did you look at the source? There is a huge amount of whitespace (206 spaces, to be exact), terminated with "1 ↵".